src/store/ChartStore.js

```diff
@@ -623,7 +648,10 @@ class ChartStore {
                 return;
             }
             this.state.restoreDrawings();
+            const { Close } = this.context.stx.currentQuote();
+            this.calculateYaxisWidth(Close);
         };
+        this.yAxiswidth = 0;
```

Comment:
on each `symbol/market` change, we re-calculate the `y-axis` width due to number